### PR TITLE
Af navbar and buttons

### PR DESCRIPTION
**Pull request by obiwanken0bi**

### CODE REVIEW

File: app/views/shared/_navbar.html.erb [hunk 1]
```diff
@@ -1,34 +1,56 @@
 <div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
   <%= link_to "#", class: "navbar-brand" do %>
     <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
-    <% end %>
+  <% end %>
 
-  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
+  <!-- <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
```
Comment (Pil0u) [inline app/views/shared/_navbar.html.erb]: To uncomment

File: app/views/shared/_navbar.html.erb [hunk 2]
```diff
@@ -1,34 +1,56 @@
...
       <% if user_signed_in? %>
-        <li class="nav-item active">
-          <%= link_to "Home", "#", class: "nav-link" %>
-        </li>
+
         <li class="nav-item">
-          <%= link_to "Messages", "#", class: "nav-link" %>
+          <%= link_to "#", class: "nav-link" do %>
+            <i class="fas fa-bell"></i>
+            <!-- <i class="far fa-bell"></i> -->
```
Comment (Pil0u) [inline app/views/shared/_navbar.html.erb]: Uncomment

File: app/views/shared/_navbar.html.erb [hunk 3]
```diff
@@ -1,34 +1,56 @@
...
+          <% end %>
         </li>
+
         <li class="nav-item dropdown">
           <%= image_tag "https://kitt.lewagon.com/placeholder/users/ssaunier", class: "avatar dropdown-toggle", id: "navbarDropdown", data: { toggle: "dropdown" }, 'aria-haspopup': true, 'aria-expanded': false %>
+          <!--<img class="avatar dropdown-toggle" id="navbarDropdown" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false" src="<%#= current_user.photo.service_url %>" />-->
```
Comment (Pil0u) [inline app/views/shared/_navbar.html.erb]: Uncomment ?

Comment (obiwanken0bi) [reply app/views/shared/_navbar.html.erb]: I was waiting for cloudinary code to be OK

**Pull request merged**